pkg/app/init.go

```
@@ -178,18 +142,24 @@ func initCommand(clicontext *cli.Context) error {
 		return errors.Errorf("build.envd already exists, use --force to overwrite it")
 	}
 
-	var buildEnvdContent []byte
 	if lang == "python" {
-		buildEnvdContent, err = initPythonEnv(buildContext)
-	} else {
-		buildEnvdContent, err = templatef.ReadFile("template/" + lang + ".envd")
+		err = InitPythonEnv(buildContext)
+		if err != nil {
+			return err
+		}
+	} else if lang == "r" {
+		startQuestion(RPackageChoice)
 	}
-	if err != nil {
-		return err
+
+	startQuestion(CudaChoice)
+	if selectionMap[LabelCudaChoice][0] == "Yes" {
+		startQuestion(CudaVersionChoice)
 	}
-	err = os.WriteFile(filePath, buildEnvdContent, 0644)
+
+	err = generateFile(clicontext)
 	if err != nil {
-		return errors.Wrapf(err, "Failed to create build.envd")
+		return nil
```

Should we return an error here?